packages/contracts/deploy-config/devnetL1-template.json

```diff
@@ -0,0 +1,65 @@
+{
+  "l1ChainID": 900,
+  "l2ChainID": 901,
+  "l2BlockTime": 2,
+  "l1StartingBlockTag": "earliest",
+  "maxSequencerDrift": 300,
+  "sequencerWindowSize": 200,
+  "channelTimeout": 120,
+  "p2pSequencerAddress": "0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc",
+  "batchInboxAddress": "0xff00000000000000000000000000000000000000",
+  "batchSenderAddress": "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC",
+  "l2OutputOracleSubmissionInterval": 4,
+  "l2OutputOracleStartingTimestamp": -1,
+  "l2GenesisBlockGasLimit": "0x1c9c380",
+  "l1BlockTime": 3,
+  "l1UseClique": true,
+  "cliqueSignerAddress": "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
+  "protocolVaultRecipient": "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
+  "l1FeeVaultRecipient": "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
+  "proxyAdminOwner": "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
+  "finalizationPeriodSeconds": 120,
+  "deploymentWaitConfirmations": 1,
+  "fundDevAccounts": true,
+  "l2GenesisBlockBaseFeePerGas": "0x1",
+  "gasPriceOracleOverhead": 2100,
+  "gasPriceOracleScalar": 1000000,
+  "enableGovernance": true,
+  "governanceTokenSymbol": "KRO",
+  "governanceTokenName": "Kroma",
+  "governanceTokenOwner": "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
+  "eip1559Denominator": 50,
+  "eip1559Elasticity": 6,
+  "l1GenesisBlockTimestamp": "0x64c811bf",
+  "l2GenesisRegolithTimeOffset": "0x0",
+  "l2GenesisSpanBatchTimeOffset": "0x0",
+  "l2GenesisCanyonTimeOffset": "0x40",
```

Comment:
I was wondering if there is a reason why it is 0x40 and not 0x00.
And for l2GenesisCanyonTimeOffset to be set, shouldn't eip1559DenominatorCanyon also be included?

Reply:
eip1559DenominatorCanyon added.

If I set the l2GenesisCanyonTimeOffset to 0x40, I think we can test the hard fork on the devnet.